lib/cc/engine/analyzers/file_list.rb

```diff
@@ -18,8 +20,8 @@ def files
 
         def matching_files
           paths.map do |glob|
-            Dir.glob("./#{glob}").reject do |f|
-              File.directory?(f)
+            Dir.glob("./#{glob}").reject do |path|
+              File.directory?(path)
```

Comment:
It already was, so we should probably keep it that way. If we're running into symlinks with engines we should look at fixing that somewhere upstream.